addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -243,6 +243,13 @@ class StudyService extends Service {
     return result;
   }
 
+  _getAllowedStudies(permissions) {
```

Comment:
minor:  How about this?
```
_getAllowedStudies(permissions = [])
```
It will guard against permissions being undefined or null

Reply:
line 270 does that already

Reply:
Added it just in case some other function calls this without that check